fetch.bs

```diff
@@ -324,6 +330,165 @@ specialized multimap. An ordered list of key-value pairs with potentially duplic
+ <var>name</var> from <var>list</var>.
+
+ <li><p>If <var>initialValue</var> is null, then return null.
+
+ <li><p>Let <var>input</var> be the result of <a>isomorphic decoding</a> <var>initialValue</var>.
+
+ <li><p>Let <var>position</var> be a <a for=string>position variable</a> for <var>input</var>,
+ initially pointing at the start of <var>input</var>.
+
+ <li><p>Let <var>values</var> be a <a for=/>list</a> of <a for=/>strings</a>, initially empty.
+
+ <li><p>Let <var>value</var> be null.
+
+ <li>
+  <p>While <var>position</var> is not past the end of <var>input</var>:
+
+  <ol>
+   <li>
+    <p>Let <var>temporaryValue</var> be the result of <a>collecting a sequence of code points</a>
+    that are not U+0022 (") or U+002C (,) from <var>input</var>, given <var>position</var>.
+
+    <p class=note><var>temporaryValue</var> might be the empty string.
+
+   <li><p>If <var>value</var> is null, then set <var>value</var> to <var>temporaryValue</var>.
+
+   <li><p>Otherwise, append <var>temporaryValue</var> to <var>value</var>.
+
+   <li>
+    <p>If <var>position</var> is not past the end of <var>input</var>, then:
+
+    <ol>
+     <li>
+      <p>If the <a for=/>code point</a> at <var>position</var> within <var>input</var> is
+      U+0022 ("), then:
+
+      <ol>
+       <li><p>Append U+0022 (") to <var>value</var>.
+
+       <li><p>Advance <var>position</var> by 1.
+
+       <li><p>Append the result of <a>collecting a sequence of code points</a> that are not
+       U+0022 (") from <var>input</var>, given <var>position</var>, to <var>value</var>.
+
+       <li>
+        <p>If <var>position</var> is not past the end of <var>input</var>, then:
+
+        <ol>
+         <li><p>Assert: the <a for=/>code point</a> at <var>position</var> within <var>input</var>
+         is U+0022 (").
+
+         <li><p>Append U+0022 (") to <var>value</var>.
+
+         <li><p>Advance <var>position</var> by 1.
+
+         <li><p><a for=iteration>Continue</a>.
+        </ol>
+      </ol>
+
+     <li>
+      <p>Otherwise:
+
+      <ol>
+       <li><p>Assert: the <a for=/>code point</a> at <var>position</var> within <var>input</var> is
+       U+002C (,).
+
+       <li><p>Advance <var>position</var> by 1.
+      </ol>
+    </ol>
+
+   <li><p>Remove all <a>HTTP tab or space</a> from the start and end of <var>value</var>.
+
+   <li><p><a for=list>Append</a> <var>value</var> to <var>values</var>.
+
+   <li><p>Set <var>value</var> to null.
+  </ol>
+
+ <li><p>Return <var>values</var>.
+</ol>
+
+<div class=example id=example-header-list-get-decode-split>
+ <p>This is how <a>get, decode, and split</a> functions in practice with `<code>A</code>` as
+ <var>name</var> argument:
+
+ <table>
+  <tr>
+   <th>Headers (as on the network)
+   <th>Output
+  <tr>
+   <td>
+    <pre><code>
+A: nosniff,
+</code></pre>
+   <td rowspan=2>« "<code>nosniff</code>", "" »
+  <tr>
+   <td>
+    <pre><code>
+A: nosniff
+B: sniff
+A:
+</code></pre>
+  <tr>
```

Comment:
Maybe add blank lines before each "real" row.  The rowspan and <pre> tags make it rather difficult to follow this table.  Or just add HTML comments between entries.

Reply:
The Preview and Diff links in OP should help here, that way you don't have to read source code but can look at rendered HTML.